src/05-derivatives/01-introduction.md

```diff
@@ -0,0 +1,219 @@
+# Goals and scope
+
+Derivatives are outputs of common processing pipelines, capturing data and
+meta-data sufficient for a researcher to understand and (critically) reuse those
+outputs in subsequent processing. Standardizing derivatives is motivated by use
+cases where formalized machine-readable access to processed data enables higher
+level processing.
+
+## Storage of derived datasets
+
+Derivatives can be stored/distributed in two ways:
+
+1.  Under a `derivatives/` subfolder in the root of the source BIDS dataset
+    folder to make a clear distinction between raw data and results of data
+    processing. Each pipeline has a dedicated directory under which it stores
+    all of its outputs. There are few restriction on the directory name; it is
+    RECOMMENDED to use the format `<pipeline>-<variant>` in cases where it is
+    anticipated that the same pipeline will output more than one variant (e.g.,
+    `AFNI-blurring`, `AFNI-noblurring`, etc.). For the sake of consistency, the
+    subfolder name MUST be a substring of `PipelineDescription.Name` field in
+    the `dataset_description.json` (see below).
+
+    For example:
+
+        <dataset>/derivatives/fmripreprocess-v1/sub-0001
+        <dataset>/derivatives/spm/sub-0001
+        <dataset>/derivatives/vbm/sub-0001
```

Comment:
The linter does not like this, but I don't think there's another way to get a properly indented code block.

Reply:
* Not supported: https://github.com/Python-Markdown/markdown/issues/53]